docs/docs/tutorial/part-4/index.mdx

```diff
@@ -420,8 +419,6 @@ const Layout = ({ pageTitle, children }) => {
 export default Layout
 ```
 
-![A screenshot of the home page in a web browser. The title of the page in the browser tab now says, "Home Page | My First Gatsby Site", and the top of the page has an unstyled paragraph that says, "My First Gatsby Site".](./index-page-with-site-title.png)
```

Comment:
Why did you remove the image?

Reply:
Ah! I think I was trying to reorder things at some point and didn't end up adding it back. Restored it now